### PR TITLE
feat: Wallet account view 4

### LINKED ISSUE #536: Wallet assets listing
### Please describe the problem that your request should resolve.
Ability to display Superhero Wallet assets

### Describe the solution you'd like
add wallet connection
add menu wallet (user) element
add subpage with assets listing and balances

### Is it already possible to achieve the same outcome in any other way? If so, how?
No

### PR DESCRIPTION
**Pull request by janmichek**
## Description
fixes #536 

There are 2 minor UX issues I will try to resolve in the following (Portfolio) issue, so I don't block the review

1. After successful wallet connection there is no loader when redirecting to connected account detail
2. When clicking on Try again, the whole page gets reloaded, which is not necessary. Only scanning again should work, but for some reason it's not working https://github.com/aeternity/aescan/issues/572

## Demo

https://github.com/aeternity/aescan/assets/15363559/64887a9e-5d48-4f4d-ad37-92793c2700ff

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] add wallet account detail

### CODE REVIEW

**Review by michele-franchi — CHANGES_REQUESTED** [review]
Generally it looks good but I would adjust the following:
1. The "Connect Wallet" button should have the same height of the network switcher:
image

`padding: var(--space-1);` on the button solves it.

2. The "Exit Wallet" dropdown should be visually similar to the menu dropdowns.
image
Example:
image
Without the min-width of the other menu items.

3. When you connect the wallet and you refresh the page the connection is lost.

File: .env.example [hunk 1]
```diff
@@ -9,6 +9,7 @@ NUXT_PUBLIC_NETWORK_NAME=MAINNET
 NUXT_PUBLIC_NETWORK_ID=ae_mainnet
 NUXT_PUBLIC_ALTERNATIVE_NETWORK_URL=http://localhost:8081
 NUXT_PUBLIC_ALTERNATIVE_NETWORK_NAME=TESTNET
+NUXT_PUBLIC_NETWORK_ID="ae_mainnet"
```
[the remark this answers is not in the document]
Comment (janmichek) [reply .env.example:12]: removed

**Comment by janmichek** [comment]
> Generally it looks good but I would adjust the following:
> 
>     1. The "Connect Wallet" button should have the same height of the network switcher:
> 
> 
> image
> 
> `padding: var(--space-1);` on the button solves it.
> 
>     2. The "Exit Wallet" dropdown should be visually similar to the menu dropdowns.
> 
> 
> image
> 
> Example:
> image
> 
> Without the min-width of the other menu items.
> 
>     3. When you connect the wallet and you refresh the page the connection is lost.

I have been discussing this with Denis. Currently is not possible to achieve this seamlessly. The app can remember last used account, but we cannot go over the user prompt to connect to the wallet again like on superhero.com
Implementation-wise, the SDK can do something to simplify it, but fundamentally will stay the same.

For now, I suggest not to implement it, wait for SDK completion, and the implement the persistency of last logged address @michele-franchi

**Comment by Liubov-crypto** [comment]
1. Fixed. 
2. 
I found the reason behind the images blocking: 

icn
img

**Comment by janmichek** [comment]
> 1. Fixed.
> 
> 
> I found the reason behind the images blocking:
> 
> icn img

Thanks, found the issue and fixed it.

**Pull request merged**